client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -311,32 +311,34 @@ export class PlanFeatures2023Grid extends Component<
 			selectedFeature,
 		} = this.props;
 		return (
-			<div className="plans-wrapper">
-				<QueryActivePromotions />
-				<div className="plan-features">
-					<div className="plan-features-2023-grid__content">
-						<div>
-							<div className="plan-features-2023-grid__desktop-view">
-								{ this.renderTable( planProperties ) }
-							</div>
-							<div className="plan-features-2023-grid__tablet-view">
-								{ this.renderTabletView() }
-							</div>
-							<div className="plan-features-2023-grid__mobile-view">
-								{ this.renderMobileView() }
+			<>
```

Comment:
This looks like the DOM structure update we reverted in https://github.com/Automattic/wp-calypso/pull/77039/commits/661333abced19f27c62b1513e861d4dbb9b23e5e. If we still agree it's not necessary, let's remove this update :)

Reply:
Ah I thought I'd done this, but it's my mistake. Totally agree with the DOM structure update. I'll update this PR now

Reply:
Rebased